bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/Capability.java

```diff
@@ -45,8 +45,11 @@
  */
 @NonNullByDefault
 public class Capability {
+    public static final int DATA_AGE_LIMIT = 1800;
```

Comment:
Please add `_S` suffix as you have done for other constants with values in seconds.

Reply:
Done